kuksa-client/Dockerfile

```diff
@@ -26,5 +26,7 @@ RUN apk add --no-cache libstdc++
 COPY --from=build /kuksa-client /kuksa-client
 ENV PYTHONUNBUFFERED=yes
 ENV GRPC_ENABLE_FORK_SUPPORT=false
+ENV PYTHONPATH=/kuksa-client
 WORKDIR /kuksa-client
-CMD ["/usr/local/bin/python", "-m", "kuksa_client"]
+CMD ["bin/kuksa-client"]
```

Comment:
I think we shall remove CMD now when we have added ENTRYPOINT, otherwise we will get error if trying to start the docker container without giving any parameters

Reply:
@erikbosch Did you use --net=host for databroker, and potentially also for the client, if only db is started with net==host note, that the IP you need to connect to is NOT 127.0.0.1 but rather the local ip of your computer, or the ip on the default docker bridge interface (often 172.17.0.1 in Linux). 

Also, in case of MAC: the databroker default port 55555 does not work there. Ever.

Reply:
Thanks for the hints, I got it working (client in docker towards native broker) in my environment by using `host.docker.internal` . 

`docker run --rm -it kuksa-client:latest --ip host.docker.internal --protocol grpc`